src/components/Login.js

```diff
@@ -9,35 +9,32 @@ class Login extends Component {
       password: ''
     };
   }
+
   handleChange(event) {
     this.setState({
       [event.target.name]: event.target.value
     })
     event.preventDefault();
   }
-  handleSubmit(event) {
-     // Connect to backend
-     alert(`E-mail: ${this.state.email}, Password: ${this.state.password}`);
-    event.preventDefault();
-  }
+
   render() {
     return (
       <div className="container">
         <h3>Login to...</h3>
         <div className="row">
-          <form className="col s12" onSubmit={event => this.handleSubmit(event)}>
+          <form className="col s12" onSubmit={(event) => this.props.onSubmit(event,this.state)}>
             {/* ---password--- */}
             <div className="row">
               <div className="input-field col s12">
-                <input id="password" type="password" name='password' className="validate" autocomplete="off"
```

Comment:
While its generally a good idea to catch errors, it would be good to keep it in different PR's ;)

Reply:
OK I will set it back and make a separate PR for it.